sygnal/gcmpushkin.py

```diff
@@ -371,10 +371,13 @@ async def _dispatch_notification_unlimited(
             body["priority"] = "normal" if n.prio == "low" else "high"
 
             for retry_number in range(0, MAX_TRIES):
+                # Should only ever be one of `to` or `registration_ids` fields
```

Comment:
Would it be clearer to rebuild the `body` at the top of this loop?

Any chance of being able to test this?